rfc/rfc004-distributed-document-format.md

```diff
@@ -111,11 +111,26 @@ The following orders are invalid:
 
 #### 3.4. Timelines
 
-Since documents are immutable, the only way to update them it by creating a new document. Subsequent versions of a document SHOULD be tracked by creating a _timeline_ using the optional **tid** and **tiv** fields. These fields SHALL NOT be used on the first document in the timeline, only on updates. The **tid** field identifies the timeline and MUST contain the reference of the first document. The **tid** field MUST be present when **tiv** is specified.
+Since documents are immutable, the only way to update them it by creating a new document.
+Subsequent versions of a document SHOULD be tracked by creating a _timeline_ using the optional **tid** and **tiv** fields.
+These fields SHALL NOT be used on the first document in the timeline, only on updates.
+The **tid** field identifies the timeline and MUST contain the reference of the first document.
```

Comment:
reference **to** the first document?

Reply:
I think both are correct, because it _is_ the reference of the first document, but we're pointing here so I like _to_ better. I'll change it.